apps/frontend/lib/api/offers.ts

```diff
@@ -0,0 +1,60 @@
+import { fetcher } from "../utils/fetcher";
+import { Action } from "redux";
+import { ThunkAction } from "redux-thunk";
+import type { InitialMainState } from "../../store/mainSlice";
+import type { InitialOffersState } from "../../store/offersSlice";
+import type { OfferWithSalary } from "@finejob/types";
+
+import { setLoading, showModal } from "../../store/mainSlice";
+import {
+  setOffers,
+  setLastOffersApiCallAddress,
+  setLoadMore,
+  setResultsPage,
+} from "../../store/offersSlice";
+
+type FetchFuncType = ThunkAction<
+  void,
+  InitialMainState & InitialOffersState,
+  unknown,
+  Action<string>
+>;
+
+export const JobsAPI = {
+  searchOffers: (path = ""): FetchFuncType => async (dispatch) => {
+    dispatch(setLoading(true));
```

Comment:
Po co pakować odpowiedzi z API do reduksa?

Reply:
A jak inaczej to zrobić, jeśli chciałbym je mieć w storze?